### ISSUE TITLE
converse: google is just repeating my command

### ISSUE THREAD

**Issue by pergolafabio** [Apipa169/Assistant-Relay-for-Hassio#27]
ok, i have this setup as addon
if i use this service: rest_command.assistant_converse
with command: tell me a joke

then my google devices are waking up, i hear then
broadcasting from xxx , tell me a joke

in other words, the google devices are just repeating my command? i was expecting that google would tell me a joke?

```
  assistant_converse:
    url: http://192.168.0.17:3000/assistant
    method: POST
    content_type: 'application/json'
    payload: '{"command":"{{ command }}", "user":"Fabio", "converse":true}'
```

**Comment by Armikas** [comment]
Same here :(

**Comment by Apipa169** [comment]
I just found out this is happening on my own set up as well. If I have time I need to test if it is only going wrong in the add-on or in Assistant Relay.

**Comment by pergolafabio** [comment]
why is this happening? because of this? this relay cant be used? how are other people using it? if such a simple command doesnt work?

**Comment by Apipa169** [comment]
It's doing the same thing locally. I have contacted the developer to see what's going on.

This is the same issue as reported a while ago in the Assistant Relay: https://github.com/greghesp/assistant-relay/issues/22

**Comment by Apipa169** [comment]
It looks like when google does not send a response it will broadcast the initial question. I think it is Google being annoying sometimes. I have sent my findings to the developer of GAR.

In this case it will broadcast your initial question:
<img width="634" alt="Schermafbeelding 2020-06-30 om 16 56 29" src="https://user-images.githubusercontent.com/44235469/86144464-fc3f9280-baf5-11ea-8a0f-6cec33813918.png">

In this case it will work fine:
<img width="635" alt="Schermafbeelding 2020-06-30 om 17 21 01" src="https://user-images.githubusercontent.com/44235469/86144557-1f6a4200-baf6-11ea-8004-e3a1f4679946.png">

**Comment by Apipa169** [comment]
https://github.com/googlesamples/assistant-sdk-nodejs/issues/13

Please give this issue a +1 and a comment that you are experiencing the same issue.

**Comment by Apipa169** [comment]
There will be a fix in GAR v4.0.
